### ISSUE TITLE
Using pre-trained model for labeling unlabeled data

### ISSUE THREAD

**Issue by mdiazm** [HuguesTHOMAS/KPConv#70]
Hi, Hugues. 
Thanks for your amazing contribution. I have been testing your code recently, but i'm not able to use pretrained model with Semantic3D dataset in a non-labeled point cloud dataset. I've been using the script test_any_model.py, but as far as I'm concerned, it is mandatory to have the training labeled data, and the whole dataset I have is unlabeled.

Is there a way to use pretrained model, e.g. Semantic3D, when it is available only test data?

Thanks a lot,
Miguel

**Comment by HuguesTHOMAS** [comment]
Hi @mdiazm,

The way the input pipeline is implemented, you have to give some labels. But they are not used during the test. So your simplest way around this problem is to give fake labels as input. Just create an array of zeros for example.

Best,
Hugues
